tools/clang/test/HLSL/ShaderOpArith.xml

```diff
@@ -2063,11 +2063,11 @@
       <Descriptor Name="U1" Kind="UAV" ResName="U1"
                   NumElements="64" StructureByteStride="4" />
       <Descriptor Name="U2" Kind="UAV" ResName="U2"
-                  NumElements="64" StructureByteStride="4" />
+                  NumElements="64" Format="R32_FLOAT" StructureByteStride="0" />
       <Descriptor Name="U3" Kind="UAV" ResName="U3" Dimension="TEXTURE1D"
                   NumElements="64" StructureByteStride="4" />
```

Comment:
In fact, I think U3 needs to be altered too. It should also have a Format="R32_FLOAT" param with zero stride. (or you can just leave that out and it defaults to zero)

Reply:
Hey Greg. I didn't have any issue with the U3 RWTexture1D case the test was passing. Would you like me to modify this patch to include changes to U3 as well?

Reply:
Sorry scratch my last comment, we were treating that case "U3" incorrectly. I made the changes you suggested to U3 and I am seeing it pass as well